src/integrationTest/resources/application-int-test-jwt-token-shared.yaml

```diff
@@ -22,5 +23,5 @@ darts-gateway:
   events:
     validate: true
   redis:
-    connection-string: redis://localhost:6379
+    connection-string: redis://localhost:6380
```

Comment:
it will work for the integration tests running aside local docker deployments which is why I made this change. To be clear they will both use separate docker instances